pages/community/index.md

```diff
@@ -7,7 +7,6 @@ sidebar: community_sidebar
 permalink: /doc/community/index.html
 ---
 
-This is where members of the LoopBack community can promote and explain their
-LoopBack-related projects.
+This is where Loopback developers can promote and explain the community projects that gives inspiration.
```

Comment:
Sorry, but this doesn't really read right.  "..gives inspiration" to who or what?  (There are some other grammatical issues as well.) 

And "inspiration" is such a subjective term and is entirely a matter of opinion.  I'd prefer to keep it neutral.

I'm open to rewording, but it should add some value and be in correct English.